File: fnc/examples/chapter10/membranemulti.m
%%
lambda = 0.5;
phi = @(r,w,dwdr) lambda./w.^2 - dwdr./r;
init = ones(301,1);
[r,w1] = bvp(phi,[0,1],[],0,1,[],init);
plot(r,w1)
xlabel('r'), ylabel('w(r)')   % ignore this line
title('Solution')   % ignore this line

%%
% By choosing a different initial guess, we arrive at another solution.
init = 0.5*ones(301,1);
[r,w2] = bvp(phi,[0,1],[],0,1,[],init);
hold on, plot(r,w2)

File: julia/scripts/chapter10/membranemulti.jl

lambda = 0.5
phi = (r,w,dwdr) -> lambda/w^2 - dwdr/r;

init = ones(301)
include("../FNC.jl")
r,w1 = FNC.bvp(phi,[0,1],[],0,1,[],init)

using Plots
plot(r,w1,xaxis=("r"),yaxis=("w(r)"),title="Solution",leg=:none)

init = 0.5*ones(301)
r,w2 = FNC.bvp(phi,[0,1],[],0,1,[],init)
plot!(r,w2)
